CONTRIBUTING.md

```diff
@@ -249,14 +249,18 @@ The process proceeds in two steps:
 1. CI prepares a release, and opens a PR with the version changes and release
    notes. These are triggered every Monday by a cron job or [manually via the
    GitHub UI][github-ui].
-2. A human reviews the PR, approves it, and merges into the trunk, at which
-   point CI kicks in to:
+   - The scheduled releases increment the patch number.
+   - Use the _Version_ input field to manually specify the version to release.
+
+   <img src="./trigger-release.png" alt="How to trigger a release" width="700px">
+2. A human reviews the PR, approves it, and **merges** (_DO NOT SQUASH OR
+   REBASE_) into the trunk, at which point CI kicks in to:
```

Comment:
Can we disable squash merge? Just asking :-)

Reply:
Sure: done!

I'll still leave this warning in place, in case the policies change in the future.